api/src/auth/jwt.js

```diff
@@ -120,8 +169,12 @@ export {
   createJWT,
   createUserTokenPayload,
   createUserJWT,
+  createUserRefreshTokenPayload,
+  createUserRefreshJWT,
   createOrgTokenPayload,
   createOrgJWT,
+  createOrgRefreshTokenPayload,
+  createOrgRefreshJWT,
   createDashboardTokenPayload,
-  createDashboardJWT
+  createDashboardJWT,
```

Comment:
Do we need to do this for dashboard tokens too @ht2?

Reply:
Good catch - very much depends if the dashboard tokens have an expiry. Given they are public and can be regenerated simply by refreshing the page, we probably don't need to; however some of the logic in this PR may change that behaviour, so it should be double checked.

Reply:
Yeah ok, @h-kanazawa can you please double check this when you go through these comments?

Reply:
I changed dashboard refresh token's expiry, but the other behaviours are not changed. Dashboard tokens are not refreshed.

Reply:
Ok thanks @h-kanazawa 👍